i18n/hi.toml

```diff
@@ -0,0 +1,138 @@
+# i18n strings for the Hindi site.
+
+[deprecation_warning]
+other = "प्रलेखन अब सक्रिय रूप से बनाए नहीं रखा गया है। वर्तमान में आप जिस संस्करण को देख रहे हैं वह एक स्थिर स्नैपशॉट है। अप-टू-डेट प्रलेखन के लिए, देखें | "
```

Comment:
`प्रलेखन पर अब सक्रिय रूप से काम नहीं किया जा रहा है।`

Can someone suggest a better word for `संस्करण`?

Can you remove `|` from the end?